app/src/main/java/com/randomappsinc/studentpicker/home/NameListsAdapter.java

```diff
@@ -146,20 +160,19 @@ public void renameList() {
         public void deleteList() {
             showDeleteDialog(position);
         }
-    }
 
-    @Override
-    public View getView(int position, View view, ViewGroup parent) {
-        NameListViewHolder holder;
-        if (view == null) {
-            LayoutInflater vi = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-            view = vi.inflate(R.layout.names_list_item, parent, false);
-            holder = new NameListViewHolder(view);
-            view.setTag(holder);
-        } else {
-            holder = (NameListViewHolder) view.getTag();
+        @Override
+        public void onClick(View v) {
+            if (listItemClickListener != null) {
+                int position = getAdapterPosition();
+                if (position != RecyclerView.NO_POSITION) {
```

Comment:
I have also personally never seen this case happen; we should remove this check. When it comes to bugs, we should fail fast. This means that if there's some weird case that should never happen, we shouldn't guard against it with an if check; we should just let it happen and crash the app. This allows us to find bugs faster instead of suppressing them and most likely leaving the user in a weird UI state which they may not report.